packages/blocks/embed/src/types.ts

```diff
@@ -13,7 +13,7 @@ export type Actions =
   | { type: "UPDATE_STATE"; payload: Partial<AppState> }
   | { type: "RESET_STATE" };
 
-export type ProviderNames =
```

Comment:
For me, `ProviderNames` was a misnomer since it wasn't an array of `ProviderName`s, but just a union of various strings.